bokehjs/test/core/util/array.coffee

```diff
@@ -5,7 +5,84 @@ array = utils.require "core/util/array"
 
 describe "array module", ->
 
+  it "copy should return a new copy", ->
+    a = [0, 1, 2]
+    expect(array.copy(a)).to.deep.equal [0, 1, 2]
```

Comment:
This test doesn't assert that a copy is new. To assert the test statement you have to test whether inplace modification of copy doesn't affect the original. This can be improved later.